client/src/components/App.js

```diff
@@ -34,10 +34,9 @@ const App = (props) => {
         <Route exact path="/" component={ShoesListPage} />
         <Route exact path="/shoes" component={ShoesListPage} />
         <Route exact path="/shoes/new" component={NewShoeForm} />
-        <Route 
-          exact 
-          path="/shoes/:id" 
-          render={props => <ShoeShowPage {...props} user={currentUser}/>}
+        <Route exact path="/shoes/:id"
+          render= {
+            (props) => <ShoeShowPage {...props} user={currentUser}/>}
```

Comment:
each prop for Route should be on a separate line
```js
<Route
  exact
  path="/shows/:id"
  render={(props) => ... }
/>
```

follow this pattern for all Routes for readability